freqtrade/optimize/hyperopt/hyperopt_optimizer.py

```diff
@@ -236,34 +237,38 @@ def init_spaces(self):
             + self.max_open_trades_space
         )
 
-    def assign_params(self, params_dict: dict[str, Any], category: str) -> None:
+    def assign_params(
+        self, backtesting: Backtesting, params_dict: dict[str, Any], category: str
```

Comment:
Did you intentionally leave this as "pass in"? 
it's an instance method - so i don't think this should create a problem if we use `self.backtesting` within?

Reply:
fixed